.storybook/preview.tsx

```diff
@@ -13,15 +13,57 @@ import './css/styleguide-only.css';
 import type { Story } from '@storybook/react';
 import React from 'react';
 
+/**
+ * From BM theming from here down
+ */
+
+let cachedTheme = 'theme-1';
+// Update the storybook toolbar with any custom buttons you want to add
+export const globalTypes = {
+  theme: {
+    name: 'Theme',
+    description: 'Brand theme',
+    defaultValue: cachedTheme,
+    toolbar: {
+      icon: 'paintbrush',
+      items: ['theme-1', 'theme-2'],
+      showName: false,
+    },
+  },
+};
+
+// Add in a decorator to render the proper theme properties
+const withThemeContainer = (Story: any, context: any) => {
+  const css =
+    require(`!css-loader!../src/design-tokens/${context.globals.theme}/tokens.css`).default;
+
+  if (!document.getElementById('ds-styles')) {
+    const style = document.createElement('style');
+    style.id = 'ds-styles';
+    document.head.appendChild(style);
+  }
+
+  // Add the CSS in to the head
+  // @ts-ignore
+  document.getElementById('ds-styles').innerHTML = css;
```

Comment:
Looks like we're:
- Generating a different CSS file for each theme
- Swapping out the content of a style tag here when changing theme

While that works, I think a better approach would be to use CSS variables. Something along the lines of https://lukelowrey.com/css-variable-theme-switcher/, where:
- Use CSS variables for tokens we want to be themable
- Set the default values on `:root`, which we do anyway
- Now define new values for the variables anywhere we want to theme them

This has a couple benefits:
- Much simpler. No need to switch between multiple stylesheets.
- The application (such as LP) can define the theme. This repo doesn't necessarily need to be aware of what the themes are.
- Uses web standards